Mixpanel alternatives (#6836)
PostHog/posthog.com: contents/blog/mixpanel-alternatives.md

- 📺 **Session replays** with event timelines, console logs, and network activity. You can also download recordings to save them beyond the standard 90-day retention policy.

- 🚩 **Feature flags** with local evaluation (for faster performance) and JSON payloads, so you can make changes to your product without deploying new code. 

- 🧪 **A/B tests** with up to 9 test variations, primary and secondary metrics. Automatically calculates recommended duration, sample size, and statistical significance.

- 💬 **Surveys**, including open text, link with custom CTA, ratings (for NPS and PMF surveys), single and multiple choice. Target users based on user properties, URL, or using existing feature flags.

### Who uses PostHog?

Typical PostHog users are:

- Product-minded engineering teams
- Data-savvy product managers
- Startups and mid-size enterprises

Customers include [AssemblyAI](/customers/assemblyai), [Hasura](/customers/hasura), [Vendasta](/customers/vendasta), and Airbus.

### How does PostHog compare to Mixpanel?

> PostHog's built-in features make it the obvious choice if you like the idea of consolidating around one platform. Mixpanel's offers plentiful third-party integrations for features it doesn't have, but this creates a messy workflow for teams.

Reviewer: ```suggestion
PostHog's built-in features make it the obvious choice if you like the idea of consolidating around one platform, and it can be extended further thanks to it being open source. Mixpanel offers plentiful third-party integrations for features it doesn't have, but this creates a messy workflow for teams. 
```

            <td>Track where users click and why</td>
        </tr>
        <tr>
            <td><strong>Open source</strong></td>
            <td className="text-center"><span className="text-green text-lg">✔</span></td>            
            <td className="text-center"><span className="text-red text-lg">✖</span></td>
            <td>Build your own apps and contribute code</td>
        </tr>
        <tr>
            <td><strong>Autocapture</strong></td>
            <td className="text-center"><span className="text-green text-lg">✔</span></td>            
            <td className="text-center"><span className="text-red text-lg">✖</span></td>
            <td>Capture events without manual instrumentation</td>
        </tr>
        <tr>
            <td><strong>Notebooks</strong></td>
            <td className="text-center">Public beta</td>            
            <td className="text-center"><span className="text-green text-lg">✔</span></td>
            <td>Create shareable notebooks for analysis</td>
        </tr>                           
    </tbody>
</table>
</div>

![posthog](../images/blog/mixpanel-alternatives/posthog.png)

### How popular is PostHog?

> According to [data compiled by Jason Packer](https://www.linkedin.com/posts/jhpacker_just-updated-the-popularity-numbers-on-my-activity-7112462135120601088-YLdh/), an independent analytics consultant, _PostHog is the fastest growing Mixpanel alternative_ in the market,

Reviewer: ```suggestion
According to [data compiled by Jason Packer](https://www.linkedin.com/posts/jhpacker_just-updated-the-popularity-numbers-on-my-activity-7112462135120601088-YLdh/), an independent analytics consultant, _PostHog is the fastest growing Mixpanel alternative_ in the market.
```

> **PostHog** is deployed by 2,365 (0.2%) of the top 1 million websites in 2023, up +327% from 554 the previous year. **Mixpanel** is deployed by 5,218 (0.5%), down -17% from 6,295 the previous year.

Reviewer: As of...

> **PostHog** is deployed by 2,365 (0.2%) of the top 1 million websites in 2023, up +327% from 554 the previous year. **Mixpanel** is deployed by 5,218 (0.5%), down -17% from 6,295 the previous year.

Author: ```suggestion
As of Jul 2023, **PostHog** is deployed by 2,365 (0.2%) of the top 1 million websites in 2023, up +327% from 554 the previous year. **Mixpanel** is deployed by 5,218 (0.5%), down -17% from 6,295 the previous year.
```

### Why do companies use PostHog?

> According to [reviews on G2](https://www.g2.com/products/posthog/reviews), companies use PostHog because:

Reviewer: ```suggestion
According to [reviews on G2](https://www.g2.com/products/posthog/reviews), companies trust PostHog because:
```

Always make it emotive, imho

> 1. **It replaces multiple tools:** PostHog can replace Mixpanel (product analytics), LaunchDarkly (feature flags and A/B testing), and Hotjar (session replay and surveys). This simplifies workflows and ensures all their data is in one place.

Reviewer: ```suggestion
1. **It replaces multiple tools:** PostHog can replace [Mixpanel](/blog/posthog-vs-mixpanel) (product analytics), LaunchDarkly (feature flags and A/B testing), and [Hotjar](/blog/posthog-vs-hotjar) (session replay and surveys). This simplifies workflows and ensures all their data is in one place.
```

2. **Pricing is transparent and scalable:** Reviewers appreciate how PostHog's pricing scales as they grow, ensuring they don't overspend. There's a [generous free tier](/pricing), and companies eligible for [PostHog for Startups](/startups) get $50k in additional free credits.

3. **They need a complete picture of users:** PostHog includes every tool necessary to understand users and improve products. This means using funnels to track conversion, watching session replays to see where users get stuck, testing solutions with A/B tests, and gathering feedback with user surveys.

> ### Bottom line 
>

> > PostHog is the best Mixpanel alternative for startups and mid-size companies, especially those who would rather use one integrated platform instead of multiple, more expensive tools. It's also the best alternative for engineers.

Reviewer: We should clarify WHY it's best for engineers

## Google Analytics 4 (GA4)

![GA4](../images/blog/mixpanel-alternatives/google-website.png)

### What is Google Analytics 4?

> Google Analytics 4 (GA4) is a marketing and product analytics tool that's tightly integrated with other Google products, such as Google Ads, BigQuery, Looker Studio, and Firebase. 

Reviewer: ```suggestion
Google Analytics 4 (GA4) is a marketing and web analytics tool that's tightly integrated with other Google products, such as Google Ads, BigQuery, Looker Studio, and Firebase. 
```

As of July 2023, [30.5% of the top 1 million websites](https://www.linkedin.com/posts/jhpacker_just-updated-the-popularity-numbers-on-my-activity-7112462135120601088-YLdh/) run GA4 – another 35.7% were still running GA3.

### How does GA4 compare to Mixpanel?

1. **Mixpanel is built for product teams:** Despite Google's overtures, GA4 still skews towards marketing and e-commerce use cases. Mixpanel is built for product teams and includes some useful collaboration features to support them.

2. **GA4 is harder to use:** Mixpanel reviewers appreciate its easy-to-use UI. GA4, in contrast, has faced criticism for its complexity. Moving away from popular "pre-baked" reports in favor of more powerful, highly customizable ones has upset existing users.

3. **Mixpanel is platform-agnostic:** While there's nothing stopping you from exporting GA4 data to non-Google platforms, it's mainly designed to play nicely with Google-owned products. Mixpanel happily integrates with a wide range or third-party tools across A/B testing, marketing automation, attribution, and beyond.

> ### Bottom line
>

> > GA4 is the best Mixpanel alternative for marketing teams who want to track ROI on campaigns.

Reviewer: I'd add here how it's suited to teams who only want to track data in specific ways too

Author: What do you mean by "specific ways" here?

Reviewer: As in, there are things you can't do in GA4 here. GA4 is the best Mixpanel alternative for marketing teams who want to track ROI data on Google ad campaigns, or simple web analytics metrics. It isn't suitable for those who want to explore deeper product metrics, or analyze data with SQL.

- **Greater focus on enterprise customers:** While both products serve large enterprise customers, Amplitude is more focused on these users. Pricing isn't transparent, for example, while Mixpanel makes a virtue of its transparent, predictable pricing, and a more generous free tier for startups.

- **Collaboration features:** While both products offer some twist on shareable notebooks, Mixpanel's collaboration features are more mature. Mixpanel chose to [sunset A/B testing and messaging](https://mixpanel.com/blog/why-were-sunsetting-messaging-and-experiments/) features in 2021 to double down on product analytics.

> ### Bottom line
>
> Amplitude is the best Mixpanel alternative for large enterprise companies who want to reduce the load on expensive, overworked data science teams.

## Heap

![heap](../images/blog/mixpanel-alternatives/heap-website.png)

### What is Heap?

Heap describes itself as a Digital Insights Platform. Unpacking that a bit, it means Heap offers both product analytics and session replay, while also supporting marketing use cases with multitouch attribution.

> Heap was acquired in September 2023 by Contentsquare, marketing and e-commerce analytics firm, and announced plans to integrate the two products. 

Reviewer: ```suggestion
Heap was acquired in September 2023 by Contentsquare, a marketing and e-commerce analytics firm, and announced plans to integrate the two products. 
```

---
title: "The most popular Mixpanel alternatives, compared"
date: 2023-09-26
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author: ["andy-vandervell"]
featuredImage: ../images/blog/posthog-alternatives/posthog-alternatives.jpeg
featuredImageType: full
category: General
tags:
  - Comparisons
---

> INTRO INTRO INTRO 

Reviewer: Needs an intro here, just fyi

### What is PostHog?

PostHog is an all-in-one platform that combines product analytics, session replay, feature flags, A/B testing, and surveys into one product. It can also replace customer data platforms (CDP), such as Segment.
 
### Key features

- **Product analytics**, including funnel insights, user paths, retention analysis, custom trends, and dynamic user cohorts. Power users can create custom insights using SQL.

- **Session replays** with event timelines, console logs, and network activity. You can also download recordings to save them beyond the standard 90-day retention policy.

- **Feature flags** with local evaluation (for faster performance) and JSON payloads, so you can make changes to your product without deploying new code. 

- **A/B tests** with up to 9 test variations, primary and secondary metrics. Automatically calculates recommended duration, sample size, and statistical significance.

- **Surveys**, including open text, link with custom CTA, ratings (for NPS and PMF surveys), single and multiple choice. Target users based on user properties, URL, or using existing feature flags. 

### How popular is PostHog?

> According to [data compiled by Jason Packer](https://www.linkedin.com/posts/jhpacker_just-updated-the-popularity-numbers-on-my-activity-7112462135120601088-YLdh/), an independent consultant:

Reviewer: Worth reaching out to Packer, for quotes, content, sources, etc?

> - PostHog is deployed by 2,365 (0.2%) of the top 1 million websites in 2023, up +327% from 554 the previous year. It's the fastest growing product analytics tool on the market.

Reviewer: I'd flip this to emphasize the strength, not the math. Improving 300% from a low number isn't impressive. Being the fastest growing tool on the market, is.

- Mixpanel is deployed by 5,218 (0.5%) of the top 1 million websites in 2023, down -17% from 6,295 the previous year.

Data correct as of July 2023.

### Why do companies use PostHog?

> According to [reviews on G2.com](https://www.g2.com/products/mixpanel/reviews), [summarized by Senja](https://senja.io/tools/review-mining/share?s=NOmOO5mS), companies use PostHog because:

Reviewer: Personally, I don't like the addition of Senja here. Better to say that reviews show that it has these three main perks, among others?

I'd also add a section on WHO uses PostHog. Emphasize ICP, and I can back it up with G2 data if needed. 

Of course, if you're going to bring G2 into the mix then it raises the question of whether you want to use their reports, badges, etc. If not, is G2 the right platform to bring up and would we be better pointing to the case studies for social proof instead? Case studies are probably a good addition here anyway.

Author: - RE: Senja, I think it's useful to show we're not just making up our own points here. I can probably de-emphasize so it's not so prominent, though.

- Will look at ways to work customer stories in, but I'm trying to keep the top-level stuff neutrally sourced.
